File: Count Odd Even/Count_Odd_Even.cpp
class Solution {
  public:
    
    vector<int> countOddEven(vector<int> &arr) {
        int odd =0,even = 0;
        for(auto i: arr) {
            if(i%2 == 0) even++;
            else odd++;
        }
        return {odd,even};
    }
};

File: Count Odd Even/Count_Odd_Even.java
class Solution {
    public int[] countOddEven(int[] arr) {
        int odd = 0, even = 0;
        for (int i : arr) {
            if (i % 2 == 0) {
                even++;
            } else {
                odd++;
            }
        }
        return new int[]{odd, even};
    }
}
